schema/schema.go

```diff
@@ -225,6 +225,16 @@ func ParseWithSpecialTableName(dest interface{}, cacheStore *sync.Map, namer Nam
 		schema.PrioritizedPrimaryField = schema.PrimaryFields[0]
 	}
 
+	// If there are multiple primary keys, the AUTOINCREMENT field is prioritized
+	if schema.PrioritizedPrimaryField == nil && len(schema.PrimaryFields) > 1 {
+		for _, field := range schema.PrimaryFields {
+			if _, ok := field.TagSettings["AUTOINCREMENT"]; ok {
```

Comment:
Can we use `field.AutoIncrement`?  What happens when tag is set to `autoIncrement:false`?

Reply:
Can this code be merged with here?
https://github.com/go-gorm/gorm/pull/6127/files#diff-eafe62eae191e88faad4c223c4de9af268c175ee68695872e175b814d7c5baa5L224

Reply:
@a631807682 Thank you for pointing this out, I have fixed the code based on your comment.
- use field.AutoIncrement instead
- merge with [#6127 (files)](https://github.com/go-gorm/gorm/pull/6127/files#diff-eafe62eae191e88faad4c223c4de9af268c175ee68695872e175b814d7c5baa5L224)
- add test for autoIncrement:false